fe/src/components/ColorMode/ColorMode.tsx

```diff
@@ -0,0 +1,16 @@
+import { FaMoon, FaSun } from 'react-icons/all';
```

Comment:
Can you use ChakraIcons for Sun and Moon, because I want to remove react-icons in the future #96